app/validators/idv/form_state_id_validator.rb

```diff
@@ -29,6 +30,17 @@ module FormStateIdValidator
                          char_list: invalid_chars.join(', '),
                        )
                      end
+      # rubocop:disable Layout/LineLength
+      validates_with UspsInPersonProofing::DateValidator,
+                     attributes: [:dob], less_than_or_equal_to: ->(_rec) {
```

Comment:
I think we need to use `greater_than` instead.
```suggestion
                     attributes: [:dob], greater_than: ->(_rec) {
```